python-sdk/tests/experiment/test_async_trace_isolation.py

```
@@ -0,0 +1,308 @@
+        captured, mock_post = _mock_http_capture()
+
+        experiment = Experiment("test-async-batch")
+        experiment.initialized = True
+
+        df = pd.DataFrame([{"q": f"Question {i}"} for i in range(6)])
+
+        async def task(row):
+            await asyncio.sleep(0.005)
+
+        with patch("httpx.post", side_effect=mock_post):
+            async for index, row in experiment.aloop(df.iterrows(), concurrency=3, total=6):
```

Ruff (0.15.10)

[warning] Loop control variable `index` not used within loop body

Rename unused `index` to `_index`

(B007)